From d62d7d842756bfd82cf19141a04b4160080b0efa Mon Sep 17 00:00:00 2001
Message-Id: <d62d7d842756bfd82cf19141a04b4160080b0efa.1367947969.git.minovotn@redhat.com>
In-Reply-To: <707b9b97153063374d2530e72c49b1499fc21af9.1367947969.git.minovotn@redhat.com>
References: <707b9b97153063374d2530e72c49b1499fc21af9.1367947969.git.minovotn@redhat.com>
From: Laszlo Ersek <lersek@redhat.com>
Date: Mon, 6 May 2013 19:27:54 +0200
Subject: [PATCH 089/114] qemu-ga: Document intentional fall through in
 channel_event_cb()

RH-Author: Laszlo Ersek <lersek@redhat.com>
Message-id: <1367868499-27603-32-git-send-email-lersek@redhat.com>
Patchwork-id: 51130
O-Subject: [RHEL-6.5 qemu-kvm PATCH v2 31/56] qemu-ga: Document intentional fall through in channel_event_cb()
Bugzilla: 952873
RH-Acked-by: Jeffrey Cody <jcody@redhat.com>
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>

For clarity, and to hush up Coverity.

Signed-off-by: Markus Armbruster <armbru@redhat.com>
Reviewed-by: Eric Blake <eblake@redhat.com>
Reviewed-by: Michael Roth <mdroth@linux.vnet.ibm.com>
Reviewed-by: Luiz Capitulino <lcapitulino@redhat.com>
Signed-off-by: Michael Roth <mdroth@linux.vnet.ibm.com>
(cherry picked from commit f5b795787864ddde1104a4f7c061dcb0e58e45c0)

Conflicts:

	qga/main.c
Signed-off-by: Laszlo Ersek <lersek@redhat.com>
---
 qemu-ga.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 qemu-ga.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/qemu-ga.c b/qemu-ga.c
index 30214a2..eb7fc7a 100644
--- a/qemu-ga.c
+++ b/qemu-ga.c
@@ -618,6 +618,7 @@ static gboolean channel_event_cb(GIOCondition condition, gpointer data)
         if (!s->virtio) {
             return false;
         }
+        /* fall through */
     case G_IO_STATUS_AGAIN:
         /* virtio causes us to spin here when no process is attached to
          * host-side chardev. sleep a bit to mitigate this
-- 
1.7.11.7

